liquibase-core/src/main/java/liquibase/change/core/AddForeignKeyConstraintChange.java

```diff
@@ -369,6 +374,7 @@ public String getConfirmationMessage() {
      * @deprecated No longer supported in 3.0
      */
     @Deprecated
+    @DatabaseChangeProperty(description = "Deprecated")
```

Comment:
This is not related to the documentation, but we should probably remove code that's been deprecated for this long.

Reply:
Actually, and this is related to the documentation, @adrian-velonis1 .  If we do not remove this code, what should the description say to use instead? For example, just above we have a deprecation with this information: 

`@deprecated Use {@link #getOnDelete()}`

Reply:
It was my mistake to not checked that the deprecated method was not being used. I can remove it if you both are ok with it.

Reply:
According to this [old post](https://forum.liquibase.org/t/addforeignkeyconstraintchange-referencesuniquecolumn-support-in-3-0-1/2298), `referencesUniqueColumn` was removed because it "wasn’t really doing anything," but was kept in the code so people's changelogs didn't break. I wouldn't mind if it were removed from the code now, as that was 10 years ago and I doubt we have many people on 2.X. Nathan might have an opinion, I'm not sure what the standard operating procedure is.

Reply:
We've tended to not remove deprecated code until we're forced to in order to keep backwards compatibility as long as possible. Even if people are on a current version of liquibase now, they may still have changesets written from back then which have those attributes in their "already applied" changesets and removing it will cause problems.